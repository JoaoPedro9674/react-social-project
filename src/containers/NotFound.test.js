import React from 'react';
import renderer from 'react-test-renderer';

import { NotFound } from './NotFound';

test('App should render correctly', () => {
  const component = renderer.create(
    <NotFound />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
