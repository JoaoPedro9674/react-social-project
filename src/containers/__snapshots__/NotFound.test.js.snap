exports[`test NotFound should render correctly 1`] = `
<div
  className="container">
  <div
    className="row middle-xs center-xs">
    <div
      className="col-xs-12"
      style={
        Object {
          "padding": "10% 0",
        }
      }>
      <img
        alt=""
        src="https://s3.amazonaws.com/learn-react/assets/not-found/404.gif" />
      <h2>
        Not found :(
      </h2>
      <button
        className="btn btn-default"
        onClick={[Function]}>
        go back home
      </button>
    </div>
  </div>
</div>
`;
